python/orca/src/bigdl/orca/ray/raycontext.py

```diff
@@ -497,149 +67,12 @@ def stop(self):
-            assert len(master_process_infos) == 1, \
-                "There should be only one ray master launched, but got {}"\
-                .format(len(master_process_infos))
-            master_process_info = master_process_infos[0]
-            redis_address = master_process_info.master_addr
-            raylet_process_infos = ray_rdd.mapPartitions(
-                self.ray_service.gen_raylet_start(redis_address)).collect()
-            raylet_process_infos = [process for process in raylet_process_infos if process]
-            assert len(raylet_process_infos) == self.num_ray_nodes - 1, \
-                "There should be {} raylets launched across the cluster, but got {}"\
-                .format(self.num_ray_nodes - 1, len(raylet_process_infos))
-            process_infos = master_process_infos + raylet_process_infos
-
-        self.ray_processesMonitor = ProcessMonitor(process_infos, self.sc, ray_rdd, self,
-                                                   verbose=self.verbose)
-        return self.ray_processesMonitor.master.master_addr
-
-    def _start_restricted_worker(self, num_cores, node_ip_address, redis_address):
-        extra_param = {"node-ip-address": node_ip_address}
-        if self.extra_params is not None:
-            extra_param.update(self.extra_params)
-        command = RayServiceFuncGenerator._get_raylet_command(
-            redis_address=redis_address,
-            ray_exec="ray",
-            password=self.redis_password,
-            ray_node_cpu_cores=num_cores,
-            object_store_memory=self.object_store_memory,
-            extra_params=extra_param)
-        modified_env = self.ray_service._prepare_env()
-        print("Executing command: {}".format(command))
-        process_info = session_execute(command=command, env=modified_env,
-                                       tag="raylet", fail_fast=True)
-        RayServiceFuncGenerator.start_ray_daemon("python",
-                                                 pid_to_watch=os.getpid(),
-                                                 pgid_to_kill=process_info.pgid)
-
-    def _start_driver(self, num_cores, redis_address):
-        print("Start to launch ray driver on local")
-        import ray._private.services
-        node_ip = ray._private.services.get_node_ip_address(redis_address)
-        self._start_restricted_worker(num_cores=num_cores,
-                                      node_ip_address=node_ip,
-                                      redis_address=redis_address)
-        ray.shutdown()
-        init_params = dict(
-            address=redis_address,
-            _redis_password=self.ray_service.password,
-            _node_ip_address=node_ip
-        )
-        if version.parse(ray.__version__) >= version.parse("1.4.0"):
-            init_params["namespace"] = "az"
-        return ray.init(**init_params)
+            raise Exception("No active RayContext. Please create a RayContext and init it first")
```

Comment:
The user won't know how to create `RayContext`; I think you should just ask the user to call `init_orca_context` first.